cluster/gce/gci/mounter/startmounter.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# start rpcbind if it is not started yet
+s=`/etc/init.d/rpcbind status`
+if [[ $s == *"not running"* ]]; then
+   echo "Starting rpcbind: /sbin/rpcbind -w"
+   /sbin/rpcbind -w
+fi
+echo `/etc/init.d/rpcbind status`
```

Comment:
If glusterfs does not rely on rpcbind (I don't think it does, but would need to check) then ideally we would only start rpcbind when someone tries to mount nfs, correct? That said, if parsing the args and figuring out what fs type the user is trying to mount is too ugly, I can live with this as-is.